<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="AMICHAITO1-HHS" name="TestNG.sanityAllOptionsTest" tests="26" failures="0" timestamp="09 Dec 2019 13:10:13 GMT" time="560.927" errors="6">
  <testcase name="initialOfferQuestions" time="17.836" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="initialOfferQuestions" time="17.929" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="callMeWizard1" time="12.984" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="callMeWizard1" time="15.573" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="initialOfferScreen" time="20.437" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="initialOfferScreen" time="20.338" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="finalOfferQuestions" time="44.688" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="finalOfferQuestions" time="42.672" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="finalOfferScreen" time="18.490" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="finalOfferScreen" time="18.878" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="covers" time="2.422" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="covers" time="2.406" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="payments" time="17.469" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="payments" time="17.451" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="returnSteps" time="35.636" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="returnSteps" time="35.464" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="forwardSteps" time="59.664" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="forwardSteps" time="59.534" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="iframeCreditGuard" time="11.190" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed in Iframe CreditGuard">
      <![CDATA[java.lang.Exception: Failed in Iframe CreditGuard
at TestNG.sanityAllOptionsTest.iframeCreditGuard(sanityAllOptionsTest.java:1251)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- iframeCreditGuard -->
  <testcase name="iframeCreditGuard" time="11.143" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed in Iframe CreditGuard">
      <![CDATA[java.lang.Exception: Failed in Iframe CreditGuard
at TestNG.sanityAllOptionsTest.iframeCreditGuard(sanityAllOptionsTest.java:1251)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- iframeCreditGuard -->
  <testcase name="thankYouPage" time="35.324" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptionsTest.thankYouPage(sanityAllOptionsTest.java:1268)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- thankYouPage -->
  <testcase name="thankYouPage" time="35.316" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptionsTest.thankYouPage(sanityAllOptionsTest.java:1268)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- thankYouPage -->
  <testcase name="registeredAccount" time="4.028" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptionsTest.registeredAccount(sanityAllOptionsTest.java:1289)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- registeredAccount -->
  <testcase name="registeredAccount" time="4.055" classname="TestNG.sanityAllOptionsTest">
    <error type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptionsTest.registeredAccount(sanityAllOptionsTest.java:1289)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- registeredAccount -->
  <testcase name="callMeWizard2" classname="TestNG.sanityAllOptionsTest">
    <skipped/>
  </testcase> <!-- callMeWizard2 -->
  <testcase name="callMeWizard2" classname="TestNG.sanityAllOptionsTest">
    <skipped/>
  </testcase> <!-- callMeWizard2 -->
</testsuite> <!-- TestNG.sanityAllOptionsTest -->
